go/vt/vtgate/resilient_srv_topo_server.go

```diff
@@ -108,7 +108,7 @@ type srvKeyspaceEntry struct {
 	keyspace string
 
 	// the mutex protects any access to this structure (read or write)
-	mutex sync.Mutex
+	mutex sync.RWMutex
 
 	insertionTime time.Time
```

Comment:
I don't think we need this any more, just having 'value != nil' is the test we should be doing to see if it's set?

Unless we display the age of the value in the status page... Then we can populate insertionTime, but I would still test value for nil instead of insertionTime IsZero...